### DISCUSSION TITLE
Auth Token Expiry

### DISCUSSION THREAD

**Discussion by idristarwala** [fastapi-users/fastapi-users#933] [Q&A] [answered]
Hi,

I am using the FastApi-Users framework to implement a front end website. The issue I am facing is that the authentication token expires in 30 mins. Is there a way to increase that time ?

Regards,
Idris

**Comment by frankie567** [comment, marked as answer]
Every authentication strategy has a `lifetime_seconds` parameter you can tune to set the lifetime of the token: https://fastapi-users.github.io/fastapi-users/configuration/authentication/strategies/jwt/
